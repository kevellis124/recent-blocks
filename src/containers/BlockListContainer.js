import React, {Component} from 'react';
import BlockList from "../components/BlockList";

class BlockListContainer extends Component {

    render() {
        return (
            <BlockList/>
        );
    }

}

export default BlockListContainer;