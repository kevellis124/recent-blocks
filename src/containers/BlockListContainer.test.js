import { shallow } from '../enzyme';
import * as React from 'react';
import BlockListContainer from './BlockListContainer';

describe('<BlockListContainer />', () => {
    describe('By default', () => {
        let component;
        beforeAll(() => {
            component = shallow(<BlockListContainer/>);
        });
        it('hasnt changed', () => {
            expect(component.html()).toMatchSnapshot();
        });
    });
});