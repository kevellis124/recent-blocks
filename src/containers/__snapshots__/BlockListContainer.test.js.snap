// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockListContainer /> By default hasnt changed 1`] = `"<div id=\\"BlockList\\"></div>"`;
